jwala-services/src/main/java/com/cerner/jwala/service/resource/impl/ResourceServiceImpl.java

```diff
@@ -1003,6 +1023,37 @@ private void doUnpack(final String hostName, final String destPath) {
         return resourceFileString;
     }
 
+    private Application setApplicationWarDeployPath(Application application) {
```

Comment:
The method does a lot of things...maybe we need to put java docs or maybe we can do something like this ?

    updatedApp = given(application).getWarResourceContent().getMetaData().tokenize().updateWarPath();

https://en.wikipedia.org/wiki/Fluent_interface